src/js/actions/ChaosActions.js

```diff
@@ -0,0 +1,104 @@
+var EventEmitter = require("events").EventEmitter;
+var lazy = require("lazy.js");
+var request = require("../helpers/qajaxWrapper");
+
+var config = require("../config/config");
+var util = require("../helpers/util");
+
+var AppsStore = require("../stores/AppsStore");
+var ChaosEvents = require("../events/ChaosEvents");
+
+function requestApp(appId, onSuccess, onError) {
+  request({
+    url: `${config.apiURL}v2/apps/${appId}`
+  })
+  .success(function (data) {
+    onSuccess(data.body.app);
+  })
+  .error(function () {
+    onError();
+  });
+}
+
+function deleteTasks(taskIds = [], onSuccess = util.noop) {
```

Comment:
Although it is [not required](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#Parameters_without_defaults_after_default_parameters), I think it's good practice to supply a default value for all arguments following optional ones.  

What do you think @aldipower ?